### PR TITLE
fix: icons positioning

### PR DESCRIPTION
**Pull request by bdrtsky**
Moving social icons & color mode switch from AppHeader to AppFooter, to leave place for Search icon and Navigation links in header (also those links/toggle is not very important to keep them in AppHeader imao)

This should also fix some Nuxt Content docs issues.

### CODE REVIEW

**Comment by Tahul** [comment]
@Atinux ; needs your inputs on this ?

I think the icons on the top right can become conflicting with other features (Search) while providing very low value to user.

I would maybe have kept the social icons (Twitter/GitHub, etc) in the header?

@R-mooon inputs would be great as well!

**Comment by bdrtsky** [comment]
I think social icons is not necessary in fixed header since they already present in 2 other places (mobile menu and footer). Being in fixed header they create almost no value (like someone will need fast access for them to go to docs Twitter page), and looks like you desperately trying to create a lead from visitor by placing social icons all over the website, even showing it to user at every moment he's looking at it (strange for open source documentation page)

**Comment by bdrtsky** [comment]
If social still necessary I would pick only GitHub icon to show in Header.

**Comment by Tahul** [comment]
I think social icons (github/twitter/others) has most value in top bar

Dark/light mode is something you usually select only once (and now is also a "global parameter" that your computer defines once and should automatically be detected by the website), I would keep it in second places like footer and mobile menu

I think we should have `social icons` + `search` slots in desktop header menu

**Comment by Tahul** [comment]
Can you update the PR to implement combination of social icons + search slot @bdrtsky ?

**Comment by Tahul** [comment]
https://github.com/nuxt/content/issues/1739

Related

**Comment by bdrtsky** [comment]
Updated @Tahul

File: components/docs/DocsPageContent.vue [hunk 1]
```diff
@@ -80,7 +80,6 @@ onBeforeUnmount(() => {
         Start writing in <ProseCodeInline>content/{{ page._file }}</ProseCodeInline> to see this page taking shape.
       </Alert>
       <template v-if="hasBody && page && bottom">
-        <!-- TODO: Finish rewrite -->
```
Comment (Atinux) [inline components/docs/DocsPageContent.vue:83]: Why removing this comment this DocsPageBottom is not done?

**Pull request merged**